matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/alias/RoomAliasAvailabilityChecker.kt

```diff
@@ -51,13 +51,13 @@ internal class RoomAliasAvailabilityChecker @Inject constructor(
         } catch (throwable: Throwable) {
             if (throwable is Failure.ServerError && throwable.httpCode == 404) {
                 // This is a 404, so the alias is available: nominal case
-                null
+                return
             } else {
                 // Other error, propagate it
                 throw throwable
             }
         }
-                ?.let {
+                .let {
```

Comment:
Strange that you had to do this change

Reply:
yes i had a very strange compilation issue where debugger was going in both if and else...

Reply:
ok...